test/selection.test.ts

```diff
@@ -389,25 +391,25 @@ describe('Selection', () => {
     });
   });
 
-  describe('join', () => {
```

Comment:
During development, an unreleased version of the API used the word 'join' for what we now call 'bind'. These tests are unchanged in function, but the word 'join' has been replaced with 'bind' for consistency.